src/lib/pickles/plonk_checks/plonk_checks.ml

```diff
@@ -393,10 +393,6 @@ module Make (Shifted_value : Shifted_value.S) (Sc : Scalars.S) = struct
               ~f:(fun i acc (s, _) -> acc * (gamma + (beta * s) + w0.(i)))
             |> negate )
       in
-      let generic =
-        let open Vector in
-        [ F.one; F.one; F.one ]
-      in
```

Comment:
I guess we're doing this to incrementally remove the generic part, but I have some trouble following this flow, as well as the wrap_domain_indicies and the override